tutordiscovery/plugin.py

```diff
@@ -115,3 +119,45 @@ def _mount_course_discovery(mounts, name):
 tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(
     list(config.get("overrides", {}).items())
 )
+
+
+@tutor_hooks.Filters.APP_PUBLIC_HOSTS.add()
+def _print_discovery_public_hosts(hosts: list[str], context_name: t.Literal["local", "dev"]) -> list[str]:
+    if context_name == "dev":
+        # todo: will may change the below dev port when i try this plugin in dev mode
+        hosts += ["discovery.{{ LMS_HOST }}:8000"]
+    else:
+        hosts += ["discovery.{{ LMS_HOST }}"]
+    return hosts
+
+
+REPO_PREFIX = "frontend-app-"
```

Comment:
Hmmmm this is a copy-paste gone wrong...

Reply:
@regis can u kindly show me an example that how can i done this in discovery/credentials. Cause I am unable to find what code of line I can change in these filters.

Reply:
Basically, what you are trying to achieve is to auto-mount the course-discovery folder every time a user runs:

    tutor mounts add /path/to/course-discovery

The code you copy-pasted from tutor-mfe will work for `frontend-app-*` folders, such as:

    tutor mounts add /path/to/frontend-app-learning

but not for course-discovery. Also, the host folder should be bind-mounted in the right location of the discovery image: not `/openedx/app`, but `/openedx/discovery` (I think).

Does that make sense?